manifest.json

```diff
@@ -10,7 +10,8 @@
 		"*://beyond20.kicks-ass.org/roll",
 		"*://app.roll20.net/editor/",
 		"*://*.dndbeyond.com/*",
-		"*://*.forge-vtt.com/game"
+		"*://*.forge-vtt.com/game",
+		"*://*.osrbeyond.com/*"
```

Comment:
Unfortunately, we can't add new things to the manifest, as it will basically disable the extension for everybody until they accept the new permissions. Last time we did that, we left a few hundred thousand users at once. Instead, we use the optional permissions system instead, and the url should be added to the `SUPPORTED_VTTS` constant.